validation/2.0/modules/ROOT/pages/validation-module-upgrade.adoc

```diff
@@ -0,0 +1,108 @@
+= Upgrading Validation Module to Version 2.0 - Mule 4
+
+Upgrade Validation module to version 2.x.
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|1.4 |2.0
+|===
+
+== Changes in This Release
+
+The following operation names changed:
+
+[%header%autowidth.spread]
+|===
+|Previous Operation Name | New Operation Name
+
+| *Is not blacklisted ip* | *Is not denied ip*
+| `is-not-blacklisted-ip` | `is-not-denied-ip`
+| *Is whitelisted ip* | *Is allowed ip*
+| `is-whitelisted-ip` | `is-allowed-ip`
+|===
+
+The following parameters names changed:
+
+[%header%autowidth.spread]
+|===
+|Previous Parameter Name | New Parameter Name
+
+| *Black list* | *Deny list*
+| `blackList` | `denyList`
+| *White list* | *Allow list*
+| `whiteList` | `allowList`
+|===
+
+[[changed_operations]]
+== Changed Operations
+
+
+=== Is Not Denied Ip
+
+`<validation:is-not-denied-ip>`
+
+Validates that an IP address is not present in the IP list.
+
+=== Is Allowed Ip
+
+`<validation:is-allowed-ip>`
+
+Validates that an IP address is present in the IP list.
+
+
+== Upgrade Prerequisites
+
+Before you perform the upgrade, you must:
+
+* Create a backup of your files, data, and configurations in case you need to restore to them.
+
+== Upgrade Steps
+
+Follow these steps to perform the upgrade:
+
+. In Anypoint Studio, create a Mule project.
+. In *Mule Palette*, click *Search in Exchange*.
+. In *Add Dependencies to Project*, enter `validation module` in the search field.
+. In *Available modules*, select *Validation module* and click *Add*.
+. Click *Finish*.
+. Verify that the `mule-validation-module` dependency version is `2.0.0` in the `pom.xml` file in the Mule project.
+
+Studio upgrades the connector automatically.
+
+
+== Post-Upgrade Steps
+
+After you install the latest version of the connector, and if you have Mule apps that use the previous operations and parameters, manually change them to the new operations and parameters:
+
+ . In Studio, drag the new operation to your flow.
+ . Configure the new operation with the same settings as the previous operation.
+ . Remove the previous operation.
+
+ You can also update the Mule app XML configuration file to the new names and parameters:
```

Comment:
Take away the space before "You can also update the Mule app XML configuration file to the new names and parameters" (the format appears different on view mode)